flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/models/NotificationDetails.java

```diff
@@ -268,6 +271,10 @@ private static void readPlatformSpecifics(
       notificationDetails.tag = (String) platformChannelSpecifics.get(TAG);
       notificationDetails.colorized = (Boolean) platformChannelSpecifics.get(COLORIZED);
       notificationDetails.number = (Integer) platformChannelSpecifics.get(NUMBER);
+      notificationDetails.audioAttributesUsage = (Integer) platformChannelSpecifics.get(AUDIO_ATTRIBUTES_USAGE);
+      if (notificationDetails.audioAttributesUsage == null) {
+        notificationDetails.audioAttributesUsage = AudioAttributes.USAGE_NOTIFICATION;
+      }
```

Comment:
Not sure if this should null checks or not, can revert if it is unnecessary

Reply:
The null checks you've added aren't necessary as the Dart/Flutter API won't allow this to happen. The issue I mentioned in my previous message can still occur so this can be reverted. I'll point out exactly where the issue I mentioned can be resolved